### ISSUE TITLE
Eslint hangs, doesn't catch 90% of the issues that cli eslint finds, and doesn't stop complaining after an issue has been resolved 

### ISSUE THREAD

**Issue by jsplaine** [AtomLinter/linter-eslint#1384]
## Issue Type
Bug

## Issue Description
eslint hangs, doesn't catch 90% of the issues that cli `eslint` finds, and doesn't stop complaining even after an issue has been resolved (like in issue #1151).
* Doesn't stop complaining even after an issue has been resolved.  
* Busy-signal shows it working on ("Current") files that have been prettied/resolved and closed. 
* It doesn't find the majority of the issues that cli `eslint` finds.  It never finds unused variables, for example.
* It never finds issues unless a change is made.  It will only find issues on that changed line, and will take several seconds to do so.

I've been dealing with this for months and have re-installed / nuked everything several times.

## Attempted resolution
* Uninstalled and re-installed all linting related packages, many times
* Uninstalled atom and deleted ~/.atom, and resinstalled Atom

Possibly unrelated, but the console shows this when I open files:

![image](https://user-images.githubusercontent.com/1927930/108564790-a6606a80-72d1-11eb-9f32-ab9bf319708c.png)

Community Packages (10) /Users/jsplaine/.atom/packages
├── busy-signal@2.0.1
├── csslint@1.2.0
├── intentions@1.1.5
├── language-javascript-jsx@0.3.7
├── linter@3.3.0
├── linter-eslint@8.5.5
├── linter-ui-default@3.2.5
├── prettier-atom@0.60.1 (disabled)
├── react@0.18.0 (disabled)
└── vim-mode@0.66.0 (disabled)

![image](https://user-images.githubusercontent.com/1927930/108567136-a4000f80-72d5-11eb-8f1b-31d116672bec.png)

.eslintrc.json 
https://pastebin.com/CTufPajF

## Bug Checklist
- [ ✔️ ] Restart Atom
- [ ✔️ ] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
- [ ✔️ ] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

## Linter Eslint: Debug output here
```
Atom version: 1.54.0
linter-eslint version: 8.5.5
ESLint version: 7.19.0
Hours since last Atom restart: 0.2
Platform: darwin
Using local project ESLint from: /Users/jsplaine/git/<private>/node_modules/eslint
Current file's scopes: [
  "source.js",
  "meta.delimiter.comma"
]
linter-eslint configuration: {
  "disabling": {
    "disableWhenNoEslintConfig": true,
    "rulesToSilenceWhileTyping": []
  },
  "global": {
    "useGlobalEslint": false,
    "eslintrcPath": "",
    "globalNodePath": ""
  },
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.flow",
    "source.babel",
    "source.js-semantic"
  ],
  "lintHtmlFiles": false,
  "autofix": {
    "fixOnSave": false,
    "ignoreFixableRulesWhileTyping": false,
    "rulesToDisableWhileFixing": []
  },
  "advanced": {
    "disableEslintIgnore": false,
    "disableFSCache": false,
    "showRuleIdInMessage": true,
    "eslintRulesDirs": [],
    "localNodeModules": ""
  }
}

```

**Comment by aminya** [comment]
Do you use a local eslint installation?

**Comment by jsplaine** [comment]
@aminya yes
eslint ```v7.19.0```

```
cat ~/.atom/config.cson
"*":
  core:
    telemetryConsent: "no"
  editor:
    fontSize: 17
  "exception-reporting":
    userId: "b6746f89-8f7c-4b99-a653-35e3d1f21f9d"
  "linter-eslint":
    disabling: {}
    global: {}
  "linter-ui-default":
    panelHeight: 172
  welcome:
    showOnStartup: false
```

**Comment by aminya** [comment]
Could you give an example of an issue that is taken by your CLI and not by linter-eslint in an open source project that we can open and try?

**Comment by aminya** [comment]
@jsplaine Unless you can provide a repository that reproduces the issue we cannot do anything here.
